StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -248,6 +258,18 @@ class RowButton: UIView {
         shadowRoundedRect.isAccessibilityElement = true
         updateAccessibilityTraits()
     }
+    
+    private func alignBadgeAndCheckmark() {
+        guard let promoBadge, let checkmarkImageView else {
+            return
+        }
+        
+        if promoBadgeConstraintToCheckmark == nil {
+            promoBadgeConstraintToCheckmark = promoBadge.trailingAnchor.constraint(equalTo: checkmarkImageView.leadingAnchor, constant: -12)
+        }
+        
+        promoBadgeConstraintToCheckmark?.isActive = isSelected
+    }
```

Comment:
If there is a better way to do this, please let me know!

Reply:
This looks fine to me!